index.html

```diff
@@ -59,7 +59,7 @@ <h2>Who are we?</h2>
             </section>
 
             <section>
-                <h3>We start from here ? </h3>
+                <h3>We start from here ?</h3>
```

Comment:
If it's a question it should be
```suggestion
                <h3>Do we start from here?</h3>
```